daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/InMemoryPrivateLedgerData.scala

```diff
@@ -37,9 +37,8 @@ private[engine] class InMemoryPrivateLedgerData extends PrivateLedgerData {
         case FrontStackCons(nodeId, nodeIds) =>
           val node = tx.nodes(nodeId)
           node match {
-            // TODO https://github.com/digital-asset/daml/issues/8020
-            case _: NodeRollback[_] =>
-              sys.error("rollback nodes are not supported")
+            case nr: NodeRollback[NodeId] =>
+              go(nr.children ++: nodeIds)
```

Comment:
Are you sure we have to consider the children of Rollback ? 
I mean the contract appearing inside a rollback have not been created and then should not be added to the state of the ledger.

Reply:
_shrug_
I can turn this back to a TODO until we understand it better.